lib/kino/js.ex

```diff
@@ -457,12 +461,22 @@ defmodule Kino.JS do
   ```
   ````
 
-  Non-binary data is automatically serialized to JSON.
+  > #### Export function {: .info}
+  >
+  > You should prefer to use the `data` argument for computing the
+  > export payload. However, if it cannot be inferred from `data`,
+  > you should just reference the original value. Do not put additional
+  > fields in `data`, just to use it for export.
   """
   @spec new(module(), term(), keyword()) :: t()
   def new(module, data, opts \\ []) do
+    # TODO: remove the old export options in Kino v0.14.0
     export =
       if info_string = opts[:export_info_string] do
+        IO.warn(
+          "passing :export_info_string to Kino.JS.new/3 is deprecated. Specify an :export function instead"
+        )
+
```

Comment:
Will we have to update many kinos or only Kino.VegaLite?

Reply:
`Kino.Maplibre` too!

Reply:
The only other one I found is [kino_wardely](https://github.com/chriseyre2000/kino_wardley).